packages/contracts-periphery/deploy/drippie/dripchecks/CheckBalanceHigh.ts

```diff
@@ -0,0 +1,19 @@
+/* Imports: External */
+import { DeployFunction } from 'hardhat-deploy/dist/types'
+
+const deployFn: DeployFunction = async (hre) => {
+  const { deterministic } = hre.deployments
+  const { deployer } = await hre.getNamedAccounts()
+
+  const { deploy } = await deterministic('CheckBalanceHigh', {
+    salt: hre.ethers.constants.HashZero,
```

Comment:
Perhaps the salt that we want to use is `keccak("drippie")` to be sure its a uniquely used salt